build/adtprobase.sh

```diff
@@ -74,4 +77,11 @@ if [ "$1x" = "headlessx" ]; then
   fi
 fi
 
-$HEADLESS"$MY_JAVA_HOME"java -Xms256m -Xmx512m "$TWEAK" $ADTPRO_EXTRA_JAVA_PARMS -cp "$ADTPRO_HOME"lib/%ADTPRO_VERSION%:"$ADTPRO_HOME""$RXTXLIB"/../RXTXcomm.jar:"$ADTPRO_HOME"lib/AppleCommander/AppleCommander-%AC_VERSION%.jar org.adtpro.ADTPro $*
+CLASSPATH="${ADTPRO_HOME}lib/ADTPro-2.0.3.jar"
+CLASSPATH+=":/usr/share/java/rxtx/RXTXcomm.jar"
+CLASSPATH+=":${ADTPRO_HOME}lib/AppleCommander/AppleCommander-1.3.5.13-ac.jar"
+
+${HEADLESS}"${MY_JAVA_HOME}"java -Xms256m -Xmx512m \
+	"$TWEAK" "${ADTPRO_EXTRA_JAVA_PARMS}" \
+	-cp "${CLASSPATH}" org.adtpro.ADTPro \
+	$*
```

Comment:
The classpath assignment and invocation isn't working on OSX - I changed it to the following, which worked:
```
CLASSPATH="${ADTPRO_HOME}"lib/%ADTPRO_VERSION%
CLASSPATH+=":/usr/share/java/rxtx/RXTXcomm.jar"
CLASSPATH+=":${ADTPRO_HOME}"lib/AppleCommander/AppleCommander-%AC_VERSION%.jar

${HEADLESS}"${MY_JAVA_HOME}"java -Xms256m -Xmx512m \
	"$TWEAK" ${ADTPRO_EXTRA_JAVA_PARMS} \
	-cp "${CLASSPATH}" org.adtpro.ADTPro \
	$*
```
Can you try with those tweaks and see if it works for you?

Reply:
Argh ... so what happened there was that I cut-and-pasted into adtprobase.sh from my live copy of adtpro.sh, noticed half of the %variable% substitution and fixed them, but missed the most important one.

Your modification works fine here.  Sorry about the lack of quality control ...

Reply:
But there's also some subtle changes with quotation placement - did you run this through a full build and verify it comes out ok for your distro?